integration_tests/sdk/conftest.py

```diff
@@ -282,22 +282,6 @@ def get_new_flow_name():
         flow_names.append(flow_name)
         return flow_name
 
-    def cleanup_flows():
-        if not pytestconfig.getoption("keep_flows"):
```

Comment:
This `--keep-flows` behavior is actually really useful during debugging. I think we're going to want to keep this! I think you'll just need to pass `pytestconfig` into `pytest_finish...` and skip the step if this is set.

Reply:
ohoh yeah good catch, I intended to move it to the `sessionfinish` fixture